tests/compatibility-tests/src/test/java/org/apache/activemq/artemis/tests/compatibility/base/ClasspathBase.java

```diff
@@ -181,7 +179,15 @@ protected ClassLoader getClasspath(String name, boolean forceNew) throws Excepti
          classPathValue = buffer.toString();
       }
 
-      Assert.assertTrue("Cannot run compatibility tests, no classpath found on ./target/" + name + ".cp", classPathValue != null && !classPathValue.trim().equals(""));
+      boolean foundClassPath = classPathValue != null && !classPathValue.trim().equals("");
+
+      if (!foundClassPath) {
+         logger.info("Rebuild project with -Pcompatibility-tests. Cannot find classpath definition for {}", name);
+      }
+
+      Assume.assumeTrue("Rebuild project with -Pcompatibility-tests.", foundClassPath);
+
+      Assume.assumeTrue("Rebuild project with -Pcompatibility-tests to be able to run this test. No classpath found on ./target/" + name + ".cp", foundClassPath);
```

Comment:
please @clebertsuconic  remove this line!

Reply:
I was going to ask earlier about the Assume's but didnt as its Draft :)

Reply:
@gemmellr I made it a draft to send fail commits and make sure tests would fail.

Reply:
Ok. Then I'll actually ask. I was confused why you changed the existing assertion check to an Assume, which will skip rather than fail the test? Originally it used an assertTrue that would fail if tripped, and then you had tried to add an explicit fail here at first, but then switched it to the [semi-duplicated] assumeTrue instead.

Reply:
@gemmellr I used to use assume before I switched to files. The idea was that if the variable wasn't set, the test for that particular class path would just be ignored.

I thought it would still be valid to have the tests running at least on the snapshots as that would at least validate the test itself.. but I can just disable it altogether if the profile is not run and keep as before. which I will do.

Reply:
@gemmellr PR has what I intend to merge now (apart from having to remove the DO-NOT-MERGE commit as I'm validating if the PR can handle the failure).

Reply:
tests are not running if I just do:

mvn -Pcompatibility-tests from ./tests/compatibility-tests


I'm fixing it.